src/System.Management.Automation/utils/PowerShellExecutionHelper.cs

```diff
@@ -1,34 +1,24 @@
-
-/********************************************************************++
-Copyright (c) Microsoft Corporation.  All rights reserved.
---********************************************************************/
```

Comment:
Because it is no longer used for completion, but for type inference.
Jason and I had a discussion, where he pointed out that for example Script Analyzer would be interested in type inference.

Previously, there was an unfortunate dependency in the AST on Completion, and this was in an effort to make it clear that the class was not only used by completion.